### ISSUE TITLE
Add support for ACI with public container registries

### ISSUE THREAD

**Issue by abhijeetgaiha** [terraform-providers/terraform-provider-azurerm#287]
Adding support for Azure Container Instance, with support for public container registries.

**Comment by abhijeetgaiha** [comment]
Working on this currently. Support for private registries to come later.

**Comment by dcaro** [comment]
I'm also looking at adding the provider for ACI

**Comment by abhijeetgaiha** [comment]
Still to come:
- Command lines
- Volume Mounts
- Environment Variables
- Private container registries

**Comment by ruzickap** [comment]
Support for Registry Arguments like it's in ```az``` command would be nice :-)

```
Image Registry Arguments
    --registry-login-server       : The container image registry login server.
    --registry-password           : The password to log in container image registry server.
    --registry-username           : The username to log in container image registry server.
```

Thanks for a great job...

**Comment by abhijeetgaiha** [comment]
@ruzickap It's on the way.

**Comment by jyapurv** [comment]
Hi, any news? :(

**Comment by jincod** [comment]
Hi @abhijeetgaiha 

Do you have any updates?

**Comment by achandmsft** [comment]
Hi @abhijeetgaiha let us know if you are working on this?

**Comment by tombuildsstuff** [comment]
👋 

Support for Image Registries in ACI was merged in #1529 which was released in v1.10 of the AzureRM Provider. You can upgrade to this version by updating the version number specified in the Provider block (as shown below) and then running `terraform init -upgrade`

```
provider "azurerm" {
  version = "=1.18.0"
}
```

Thanks!
